api/src/db/db.service.ts

```diff
@@ -67,8 +114,21 @@ export class DbService {
      * Get a document by ID
      * @param id - document ID (_id field)
      */
-    getDoc(id: string) {
-        return this.db.get(id);
+    getDoc(docId: string) {
```

Comment:
Since this is a public method, I suggest adding a test for it, especially the cases in the `catch`.

Reply:
Tests added